pkg/sensors/tracing/generictracepoint.go

```diff
@@ -430,6 +436,66 @@ func (tp *genericTracepoint) EventConfig() (api.EventConfig, error) {
 	return config, nil
 }
 
+// ReloadGenericTracepointSelectors will reload a tracepoint by unlinking it, generating new
+// selector data and updating filter_map, and then relinking the tracepoint.
+//
+// This is intentended for speeding up testing, so DO NOT USE elswhere without checking its
```

Comment:
By "do not use elsewhere" you mean outside of tracepoints or outside of `TestTracepointSelectors` (that seems to be the only use case for the next commit). 

Maybe it would be good to elaborate more on the limitations (as you said the config map is not updated, but there are other known limitation?). From my understanding this will be the skeleton to use this functionality for other than the tracepoint events.

Reply:
> By "do not use elsewhere" you mean outside of tracepoints or outside of TestTracepointSelectors (that seems to be the only use case for the next commit).

I meant outside of testing :)

> Maybe it would be good to elaborate more on the limitations (as you said the config map is not updated, but there are other known limitation?). From my understanding this will be the skeleton to use this functionality for other than the tracepoint events.

I'm not aware of other limitations, but I haven't thought it through for use-cases other than testing (e.g., reloading the sensors if the CR changes). And yes, the same approach could be use for kprobes. (Hoping to do that in a next PR.)